apisix/admin/init.lua

```diff
@@ -286,9 +286,22 @@ local function post_reload_plugins()
 end
 
 
-local function sync_local_conf_to_etcd()
-    core.log.warn("sync local conf to etcd")
+local function plugins_eq(old, new)
```

Comment:
Can we also compare `plugin_attr` for equality? Like this: 
https://github.com/apache/apisix/blob/9a40fd7cb497974f9d7bd75f128eed5dd238cdc6/apisix/plugin.lua#L147-L153

Reply:
We can't do it, as the attribute is not stored in the etcd.

Reply:
Got it.